UnityProject/Assets/Scripts/Managers/MatrixManager/MatrixManager.cs

```diff
@@ -60,20 +60,9 @@ public static MatrixManager Instance
 
 	private IEnumerator WaitForLoad()
 	{
-		float waitingTime = 0f;
 		while (Instance.spaceMatrix == null || Instance.mainStationMatrix == null)
```

Comment:
It's got a yield return so it should be fine,  basically doing an if statement every frame